packages/sampler/stories/qa/Scroller.js

```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
 	}
 }
 
+class ScrollerWithSpotlightTargetCalculation extends React.Component {
+	render () {
+		return (
+			<div>
+				<Panel>
+					<Header title="Scroll Intent (Scroller)">
+						<Button>hello</Button>
+					</Header>
+					<Scroller
+						focusableScrollbar
+						data-webos-voice-focused
+					>
+						<Repeater
```

Comment:
minor nit: Just for the cleanliness-sake for the import section of the code, let's use `Group` instead as in the other samples.